components/movementsensor/adxl345/adxl345.go

```diff
@@ -251,6 +251,19 @@ func (adxl *adxl345) LinearVelocity(ctx context.Context, extra map[string]interf
 	return r3.Vector{}, movementsensor.ErrMethodUnimplementedLinearVelocity
 }
 
+func (adxl *adxl345) LinearAcceleration(ctx context.Context, extra map[string]interface{}) (r3.Vector, error) {
```

Comment:
Now that we've implemented `LinearAcceleration` in drivers like these, do these drivers still need to implement readings? Some of them do because they supply information we haven't standardized (like temperature), but otherwise it feels like the readings are now sufficiently handled by the client?

Reply:
My understanding is that every sensor implements `Readings()` to give back all possible readings, and they should continue to do so regardless of what more specialized functions they might also implement.